backend/api/student/cohorts.py

```
@@ -34,9 +34,13 @@ async def get_student_cohorts(
         # Get professor info
         professor = db.query(User).filter(User.id == cohort.created_by).first()
         
-        # Get simulation count for this cohort
-        simulation_count = db.query(CohortSimulation).filter(
-            CohortSimulation.cohort_id == cohort.id
+        # Get simulation count for this cohort (only published/active simulations)
+        simulation_count = db.query(CohortSimulation).join(
+            Scenario, CohortSimulation.simulation_id == Scenario.id
+        ).filter(
+            CohortSimulation.cohort_id == cohort.id,
+            Scenario.is_draft == False,  # Only count published simulations
```

Ruff (0.14.0)

Avoid equality comparisons to `False`; use `not Scenario.is_draft:` for false checks

Replace with `not Scenario.is_draft`

(E712)